pkg/webhook/nad/mutator.go

```diff
@@ -0,0 +1,66 @@
+package nad
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	cniv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+	"github.com/yaocw2020/webhook/pkg/types"
+	admissionregv1 "k8s.io/api/admissionregistration/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/harvester/harvester-network-controller/pkg/network/iface"
+	"github.com/harvester/harvester-network-controller/pkg/utils"
+)
+
+type nadMutator struct {
+	types.DefaultMutator
+}
+
+var _ types.Mutator = &nadMutator{}
+
+func NewNadMutator() *nadMutator {
+	return &nadMutator{}
+}
+
+func (n *nadMutator) Create(request *types.Request, newObj runtime.Object) (types.Patch, error) {
+	netAttachDef := newObj.(*cniv1.NetworkAttachmentDefinition)
+
+	labels := netAttachDef.Labels
+	if labels == nil {
+		labels = make(map[string]string)
+	}
+
+	netconf := &utils.NetConf{}
+	if err := json.Unmarshal([]byte(netAttachDef.Spec.Config), netconf); err != nil {
+		return nil, fmt.Errorf(createErr, netAttachDef.Namespace, netAttachDef.Name, err)
+	}
+	labels[utils.KeyVlanLabel] = strconv.Itoa(netconf.Vlan)
+
+	lenOfBrName := len(netconf.BrName)
+	if lenOfBrName < 3 {
```

Comment:
`3` looks to be `len(iface.BridgeSuffix)`, it's better to also define a constant in iface, replaces the occurances in mutator, validator